public/src/js/models/collections/article.js

```diff
@@ -309,7 +309,7 @@ export default class Article extends DropTarget {
             mediator.emit(
                 'ui:open',
                 _.chain(this.editors())
-                 .filter(function(editor) { return editor.type === 'text' && editor.displayEditor(); })
+                 .filter(function(editor) { return (editor.type === 'text' || editor.type === 'text-image') && editor.displayEditor(); })
```

Comment:
how about `editor.indexOf('text') === 0`? We won't have to change it if we add more text based editors